src/platform/plugins/shared/chart_expressions/expression_xy/public/components/annotations.tsx

```diff
@@ -399,3 +400,39 @@ export const Annotations = ({
     </>
   );
 };
+
+const styles = {
+  xyAnnotationTooltip: css({
+    borderRadius: '0 !important',
+  }),
+  xyAnnotationTooltipRows: css({
+    // maxHeight: '60vh',
+    overflowY: 'hidden',
+  }),
+  xyAnnotationTooltipRow: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      fontWeight: euiTheme.font.weight.regular,
+      padding: `${euiTheme.size.s} ${euiTheme.size.m} ${euiTheme.size.s} ${euiTheme.size.s}`,
+      borderRadius: '0 !important',
+    }),
+  xyAnnotationTooltipExtraFields: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      color: euiTheme.colors.darkShade,
+      marginTop: euiTheme.size.s,
+    }),
+  xyAnnotationTooltipExtraFieldsKey: css({
+    overflowWrap: 'anywhere',
+    hyphens: 'auto',
+  }),
+  xyAnnotationTooltipExtraFieldsValue: css({
+    textAlign: 'right',
+    overflowWrap: 'anywhere',
+    hyphens: 'auto',
+  }),
+  xyAnnotationTooltipSkippedCount: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      position: 'relative',
+      textAlign: 'right',
+      fontWeight: euiTheme.font.weight.regular,
+    }),
+};
```

Comment:
I don't think it makes sense to keep the whole prefix-y name. The styles are collocated so it's ok to simplify:

```suggestion
const styles = {
  tooltip: css({
    borderRadius: '0 !important',
  }),
  rows: css({
    // maxHeight: '60vh',
    overflowY: 'hidden',
  }),
  row: ({ euiTheme }: UseEuiTheme) =>
    css({
      fontWeight: euiTheme.font.weight.regular,
      padding: `${euiTheme.size.s} ${euiTheme.size.m} ${euiTheme.size.s} ${euiTheme.size.s}`,
      borderRadius: '0 !important',
    }),
  extraFields: ({ euiTheme }: UseEuiTheme) =>
    css({
      color: euiTheme.colors.darkShade,
      marginTop: euiTheme.size.s,
    }),
  extraFieldKey: css({
    overflowWrap: 'anywhere',
    hyphens: 'auto',
  }),
  extraFieldValue: css({
    textAlign: 'right',
    overflowWrap: 'anywhere',
    hyphens: 'auto',
  }),
  skippedCount: ({ euiTheme }: UseEuiTheme) =>
    css({
      position: 'relative',
      textAlign: 'right',
      fontWeight: euiTheme.font.weight.regular,
    }),
};
```

Reply:
Ok! I've only removed the `xyAnnotation` prefix [here](https://github.com/elastic/kibana/pull/209074/commits/0eb99555c7aa501ca3a3277334aeb04bf815fa4b) since the first style refers to the tooltip and the rest are related. Let me know if it makes sense too